stable/kubernetes-external-secrets/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+name: kubernetes-external-secrets
+version: 1.0.1
+appVersion: 1.2.0
```

Comment:
App version seems to be 1.3.0 as thats the image tag thats used in the values :)

Reply:
fixed, thanks for pointing this out